atomics/T1087/T1087.yaml

```diff
@@ -13,11 +13,14 @@ atomic_tests:
     output_file:
       description: Path where captured results will be placed
       type: Path
-      default: ~/loot.txt
+      default: /tmp/T1087.txt
   executor:
     name: sh
     command: |
-      cat /etc/passwd > #{output_file}
+      if [ -f /etc/passwd ]; then cat /etc/passwd > #{output_file}; fi;
```

Comment:
Is it problematic if we run the command on a file that doesn't exist? A script or attacker may try to cat the file without checking that it exists and we may still be able to trigger on the attempt. In this case, the test may be better left to run even without checking if the file exists???

Reply:
my idea was to limit the errors running a tests if the files don't exist. I'll revert the change.